client/src/components/payment/payment-modal.vue

```diff
@@ -176,3 +175,11 @@ export default class PaymentModal extends Mixins(
   }
 }
 </script>
+
+<style lang="scss">
+@import "@/assets/scss/overrides";
+
+.close-icon {
+  float:right;
+}
```

Comment:
If it simplifies the HTML, you can use class="float-right" instead of this.

https://vuetifyjs.com/en/styles/float/#classes